pages/linux/xdg-settings.md

```diff
@@ -0,0 +1,24 @@
+# xdg-settings
+
+> Manage settings of XDG-compatible desktop environments.
+> More information: <https://portland.freedesktop.org/doc/xdg-settings.html>.
+
+- Print the default web browser:
+
+`xdg-settings get {{default-web-browser}}`
+
+- Set the default web browser to Firefox:
```

Comment:
```suggestion
- Set the default web browser to `Firefox`:
```

Reply:
I think we should better clarify where to use the ``` ` ` ``` syntax. Because I wouldn't consider `Firefox` to be technical wording. Maybe for command names only?

Reply:
Nice idea. 👍🏻

Reply:
Yes, I agree. It sometimes is not entirely clear if we should `"` or a "`"

Reply:
It should be written in style guide.